Practicum/2_week_solutions.cpp

```diff
@@ -0,0 +1,341 @@
+    //         std::cout << (char)(letter + 32) << '\n';
+    //     }
+        
+
+    //     switch (letter) {
+    //         case 'a':
+    //         case 'e':
+    //         case 'i':
+    //         case 'o':
+    //         case 'u':
+    //         case 'A':
+    //         case 'E':
+    //         case 'I':
+    //         case 'O':
+    //         case 'U':
+    //             std::cout << "Vowel";
+    //             break;
+
+    //         default:
+    //             std::cout << "Consonant";
+    //             break;
+    //     }
+
+
+    // } else {
+    //     std::cout << "Invalid character";
+    // }
+    
+
+
+    //7
+    
+    // int n1, n2, n3;
+    // std::cout << "Introduce 3 numbers: ";
+    // std::cin >> n1 >> n2 >> n3;
+
+    // int aux;
+    // int i = 0;
+
+    // while (i<2) {
+    //     if (n1 > n2) {
+    //     aux = n1;
+    //     n1 = n2; 
+    //     n2 = aux;
+    //     }
+
+    //     if (n2 > n3) {
+    //         aux = n2;
+    //         n2 = n3;
+    //         n3 = aux;
+    //     }
+
+    //     i++;
+    // }
+    
+    // std::cout << n1 << " " << n2 << " " << n3;
+
+    //8
+
+    // int radius, x , y;
+    // std::cout << "Introduce the radius of a circle and the coordinates of a point (x; y): ";
+    // std::cin >> radius >> x >> y;
+
+    // int distance = pow(x - 0, 2) + pow(y - 0, 2);
+
+    // if (distance < pow(radius, 2)) {
+    //     std::cout << "In the circle";
+    // } else if (distance > pow(radius, 2)) {
+    //     std::cout << "Out of the circle";
+    // } else {
+    //     std::cout << "On the circle";
+    // }
+    
+    //9
+
+    // float bound1, bound2, n;
+    // const float EPSILON = 0.0001;
+
+    // std::cout << "Introduce 3 numbers: ";
+    // std::cin >> bound1 >> bound2 >> n;
+
+    // if (n > bound1 && n < bound2) {
+    //     std::cout << "The number is in the interval. The interval is open.";
+    // } else if (fabs(bound1 - n) < EPSILON || fabs(bound2 - n) < EPSILON) {
+    //     std::cout << "The number is in the interval. The interval is closed.";
+    // } else {
+    //     std::cout << "The number isn't in the interval.";
+    // }
+    
+    //10
+
+    // int originalNum;
+    // std::cin >> originalNum;
+
+    // int dig1 = originalNum / 1000;
+    // int dig2 = (originalNum % 1000) / 100;
+    // int dig3 = (originalNum % 100) / 10;
+    // int dig4 = originalNum % 10;
+
+    // int num1 = dig1 * 10 + dig4;
+    // int num2 = dig2 * 10 + dig3;
+
+    // if (num1 < num2) {
+    //     std::cout << num1 << " < " << num2;
+    // } else if (num1 > num2) {
+    //     std::cout << num1 << " > " << num2;
+    // } else {
+    //     std::cout << num1 << " == " << num2;
+    // }
+
+    //11
+
+    // int day, month, year;
+    // std::cin >> day >> month >> year;
+
+    // bool leapYear = year % 4 == 0 && (year % 100 != 0 || year % 400 == 0);
+    // bool validDate = year >= 0 ? 1:0;
+
+    // switch (month) {
+    //     case 1:
```

Comment:
Combine the cases 1,3,5... in one, also the months with the 30 days, don't copy paste code 5 times.